### PR TITLE
Various changes to load testing machinery to support v17

### PR DESCRIPTION
**Pull request by rosstimothy**

### CODE REVIEW

**Review by r0mant — APPROVED** [review]

File: assets/loadtest/azure/main.tf [hunk 1]
```diff
@@ -52,6 +52,7 @@ provider "kubectl" {
 }
 
 provider "azurerm" {
+  subscription_id = vars.subscription_id
```
Comment (GavinFrazar) [inline assets/loadtest/azure/main.tf:55]:
```suggestion
  subscription_id = var.subscription_id
```

File: assets/loadtest/azure/variables.tf [hunk 2]
```diff
@@ -27,8 +27,15 @@ variable "deploy_teleport" {
...
   type        = string
   nullable    = false
   description = "Version of Teleport"
 }
+
+variable "subscription_id" {
+  type        = string
+  nullable    = false
+  description = "The Azure subscription_id"
+}
```
Comment (GavinFrazar) [inline assets/loadtest/azure/variables.tf:41]: any reasonable default or something we should put in terraform.tfvars?

File: assets/loadtest/cluster/aws/cluster.yaml [hunk 3]
```diff
@@ -4,6 +4,8 @@ metadata:
   name:
   region:
   version: "1.29"
+  tags:
+    teleport.dev/creator:
```
Comment (GavinFrazar) [inline assets/loadtest/cluster/aws/cluster.yaml:8]: if this works great, but could we put "creator: loadtest" or add a terraform variable for creator email?

File: assets/loadtest/cluster/aws/cluster.yaml [hunk 4]
```diff
@@ -16,5 +18,7 @@ addons:
 managedNodeGroups:
   - name:
     instanceType: m5.4xlarge
-    minSize: 2
+    minSize: 3
     maxSize: 10
+    tags:
+      teleport.dev/creator:
```
Comment (GavinFrazar) [inline assets/loadtest/cluster/aws/cluster.yaml:24]: likewise as above

**Review by GavinFrazar — APPROVED** [review]
Not sure if these were skipped intentionally or not since you changed the assets uri in several other places, FWIW:

```
$ rg "public.ecr.aws/gravitational/"                                                                                                                                                      (dev) 
loadtest/helm/node-agent/values.yaml
9:  repository: public.ecr.aws/gravitational/teleport-ent-distroless-debug

loadtest/helm/soaktest/values.yaml
2:  repository: public.ecr.aws/gravitational/teleport-ent-distroless-debug

$ rg "charts.releases.teleport"                                                                                                                                                           (dev) 
loadtest/control-plane/init.sh
7:helm repo add teleport https://charts.releases.teleport.dev
```

**Pull request merged**